app/models/helpers/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
```

Comment:
We can call the single-method modules Helpers, e.g. DateHelper, NameHelper etc. - that's just semantics. What I'm opposed to is a single Helpers module that ends up holding all accessory methods.